main.go

```diff
@@ -148,17 +288,17 @@ func wrapMain() error {
 	}
 
 	data := map[string]interface{}{
-		// http://readme.drone.io/usage/variables/#string-interpolation:2b8b8ac4006be88c769f5e3fd99b009a
 		"BUILD_NUMBER": build.Number,
 		"COMMIT":       build.Commit,
 		"BRANCH":       build.Branch,
-		"TAG":          "", // How?
+		"TAG":          build.Tag,
 
 		// https://godoc.org/github.com/drone/drone-plugin-go/plugin#Workspace
-		"workspace": workspace,
-		"repo":      repo,
-		"build":     build,
-		"system":    system,
+		// TODO do we really need these?
```

Comment:
do we really need this section in the `data` structure? I'm not sure what this is used for, and it didn't come up in my basic manual testing.